.editorconfig

```diff
@@ -13,5 +13,5 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 translate_tabs_to_spaces = true
-indent_style = space
+indent_style = tab
 indent_size = 4
```

Comment:
> when indent_style is set to tab, it may be desirable to leave indent_size unspecified so readers may view the file using their preferred indentation width

from http://editorconfig.org/#file-format-details

Reply:
Ran into issues with that, I think - most likely because of the asterisk. *all* files have an indent size of 2...need to override to say something else. We could remove the asterisk, I suppose. Basically saying we only want these specific file types to have these specific qualities. I tried doing something like this, and it didn't seem to work very well - but it could just be my editor + plug-in interaction (tested in both Sublime and VSC, with same unexpected results):

```
[*]
charset = utf-8
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true

[*.json, *.js]
indent_style = space
indent_size = 2

[*.md, !test/*md]
indent_style = tab
```

For some reason, when I tried various permutations of this inheritance stack I always got spaces in MD files...?? Could just be something in my environment, though it's not obvious to me what.

Reply:
At one point, I was also tempted to just do this:

```
[*.json, *.js]
charset = utf-8
end_of_line = lf
insert_final_newline = true
indent_style = space
indent_size = 2
```

And call it a day...literally only effect JS and JSON. Not sure what would have things like YML though. Also not sure how broad our lint rules go either. Would like to get this to match those as much as possible where possible - just not a deal-breaker if they don't or can't.